### ISSUE TITLE
At values slightly above the minimum to cast Life Tap, the spell casts without fizzling but does not cost health or restore mana

### ISSUE THREAD

**Issue by steak-house** [SunstriderEmu/BugTracker#2645]
**Describe the bug**
See title.

**To Reproduce**
1. With a character with decent gear (never experienced this bug until I hit 60), try to cast Life Tap below the health threshold until it succesfully casts
2. Observe that your health is not reduced, and that your mana is not restored

https://classic.wowhead.com/spell=1454/life-tap

**Expected behavior**
The spell should cast, and mana should be restored

**Additional context**
It looks like spell damage and talents increases not only the mana restored, but also the life cost of Life Tap. I do not know if that is intentional, but if so, the spell should Fizzle.

**Comment by blblblaz** [comment]
you are correct, the "Fizzled" mechanic works if you don't equip any gear, however it doesn't account for the spell damage scaling of Life Tap, thanks for the report
